index.js

```diff
@@ -60,6 +60,23 @@ function requestAsEventEmitter(opts) {
 			});
 		});
 
+		// this method is not documented, prints a warning if used while
```

Comment:
It is documented for `ServerResponse`, not for `ClientRequest` even if they both inherit from the same internal class`OutgoingMessage`.